rasa_core/tracker_store.py

```diff
@@ -31,6 +31,20 @@ def __init__(self, domain, event_broker=None):
         self.domain = domain
         self.event_broker = event_broker
 
+    @classmethod
```

Comment:
I think `staticmethod` should also be fine since we don't require access to any attributes / state of the class.

Reply:
okay, i can change that